### ISSUE TITLE
throws errors

### ISSUE THREAD

**Issue by folkertvanheusden** [jdeast/pyncraft#1]
Hi,

I've installed spigot-1.20.1 with FruitJuice-0.2.0.jar
Unfortunately everything I try fails:

```folkert@mc-spigot:~$ python3
Python 3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from pyncraft.minecraft import Minecraft
>>> mc = Minecraft.create(address="localhost", port = 4711)
 ('Running Python version:3.11.2 (main, Mar 13 2023, 12:18:29) [GCC 12.2.0]',) 
>>> print(mc)
<pyncraft.minecraft.Minecraft object at 0x7f015fca3d10>
>>> mc.setBlock(0, 100, 0, 'DANDELION')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/folkert/.local/lib/python3.11/site-packages/pyncraft/minecraft.py", line 247, in setBlock
    self.conn.send(b"world.setBlock", x, y, z, block, direction,face)
  File "/home/folkert/.local/lib/python3.11/site-packages/pyncraft/connection.py", line 39, in send
    s = b"".join([f, b"(", flatten_parameters_to_bytestring(data), b")", b"\n"])
                           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/folkert/.local/lib/python3.11/site-packages/pyncraft/util.py", line 10, in flatten_parameters_to_bytestring
    return b",".join(map(_misc_to_bytes, flatten(l)))
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/folkert/.local/lib/python3.11/site-packages/pyncraft/util.py", line 5, in flatten
    if isinstance(e, collections.Iterable) and not isinstance(e, str):
                     ^^^^^^^^^^^^^^^^^^^^
AttributeError: module 'collections' has no attribute 'Iterable'
>>> 
```

What could it be I'm doing wrong? Should I use python2 maybe?

**Comment by folkertvanheusden** [comment]
It is easy to fix:

replace pyncraft/util.py by:

```
import collections

try:
    collectionsAbc = collections.abc

except AttributeError:
    collectionsAbc = collections

def flatten(l):
    for e in l:
        if isinstance(e, collectionsAbc.Iterable) and not isinstance(e, str):
            for ee in flatten(e): yield ee
        else: yield e

def flatten_parameters_to_bytestring(l):
    return b",".join(map(_misc_to_bytes, flatten(l)))

def _misc_to_bytes(m):
    """
    Convert an arbitrary object into a string encoded as a UTF8 series of bytes.

    See `Connection.send` for more details.
    """
    return str(m).encode("utf8")
```

**Comment by jdeast** [comment]
Thanks! It looks like iterable was removed from collections in python 3.10 (I've done my development using 3.9). Your fix should work, which I'll incorporate/test/push tonight.

https://stackoverflow.com/questions/72032032/importerror-cannot-import-name-iterable-from-collections-in-python

**Comment by jdeast** [comment]
Thanks again! 

Fixed in version 0.2.1, now on github and pip
